docs/sources/panels/queries.md

```diff
@@ -82,25 +82,25 @@ Panel data source query options:
 
-- **Min interval -** Sets a minimum limit for the automatically calculated interval, typically the minimum scrape interval. If a data point is saved every 15 seconds, then there's no point in having an interval lower than that. Another use case is set it to a higher minimum than the scrape interval to get more coarse-grained, well-functioning queries.
+- **Min interval -** Sets a minimum limit for the automatically calculated interval, typically the minimum scrape interval. If a data point is saved every 15 seconds, then there's no point in having an interval lower than that. Another use case is, set it to a higher minimum than the scrape interval to get more coarse-grained, well-functioning queries.
   
 - **Interval -** The interval is a time span that you can use when aggregating or grouping data points by time. 
   
   Grafana automatically calculates an appropriate interval and it can be used as a variable in templated queries. The variable is either in seconds: `$__interval` or in milliseconds: `$__interval_ms`. It is typically used in aggregation functions like sum or average. For example, a Prometheus query using the interval variable: `rate(http_requests_total[$__interval])`.
 
-  This automatic interval is calculated based on the width of the graph. If the user zooms out a lot then the interval becomes greater, resulting in a more coarse grained aggregation whereas if the use zooms in then the interval decreases resulting in a more fine grained aggregation.
+  This automatic interval is calculated based on the width of the graph. If the user zooms out a lot then the interval becomes greater, resulting in a more coarse grained aggregation whereas if the user zooms in then the interval decreases resulting in a more fine grained aggregation.
 
   For more information, refer to [Global variables]({{< relref "../variables/global-variables.md" >}}).
 
 - **Relative time -** You can override the relative time range for individual panels, causing them to be different than what is selected in the dashboard time picker in the top right corner of the dashboard. This allows you to show metrics from different time periods or days on the same dashboard.
   
 - **Time shift -** The time shift function is another way to override the time range for individual panels. It only works with relative time ranges and allows you to adjust the time range. 
   
-  For example, you could shift the time range for the panel to be two hours earlier than the dashboard time picker. For more information, refer to [Time range controls]({{< relref "../dashboards/time-range-controls.md" >}}).
+  For example, you can shift the time range for the panel to be two hours earlier than the dashboard time picker. For more information, refer to [Time range controls]({{< relref "../dashboards/time-range-controls.md" >}}).
```

Comment:
Please undo this change.

Reply:
Can makes a better read as we are referring to something in present, could is generally used while referring to past .

Reply:
Sorry, but I think the original language is fine.